python/cloud-run-python-hello-world/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+// launch.json contains information needed for the debugger to attach to  processes
+// See https://code.visualstudio.com/Docs/editor/debugging#_launch-configurations
+// for documentation about how to create and manage configurations
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Run on Kubernetes",
+      "type": "cloudcode.kubernetes",
+      "request": "launch",
+      "skaffoldConfig": "${workspaceFolder}/skaffold.yaml",
+      "watch": true,
+      "cleanUp": true,
+      "portForward": true
+    },
+    {
+      "name": "Debug on Kubernetes",
+      "type": "cloudcode.kubernetes",
+      "request": "attach",
+      "podSelector": {
+        "app": "python-hello-world"
+      },
+      "localRoot": "${workspaceFolder}/src",
```

Comment:
A heads up that we'll need to make changes in launch.json to accomodate debugging Cloud Run container on minikube. We don't have the details yet but this will definitely change.